src/app/pages/server-compare/server-compare.component.ts

```diff
@@ -135,14 +138,44 @@ export class ServerCompareComponent implements OnInit, AfterViewInit {
 
   selectedSSLAlgo = this.availableSSLAlgos[5];
 
-  dropdownThreadCount: any;
-  availableThreadCount = [
-    { name: 'Threads per CPU: 1', value: 1 },
-    { name: 'Threads per CPU: 2', value: 2 },
-    { name: 'Threads per CPU: 4', value: 4 },
+  dropdownConnections: any;
+  connectionsOptions = [
+    { name: 'Connections: 2', value: 2 },
+    { name: 'Connections: 4', value: 4 },
+    { name: 'Connections: 8', value: 8 },
+    { name: 'Connections: 16', value: 16 },
+    { name: 'Connections: 32', value: 32 },
+    { name: 'Connections: 64', value: 64 },
+  ];
+
+  selectedConnections = this.connectionsOptions[0];
+
+  // ath the moment only SET operation is used
+  dropdownOperation: any;
+  operationsOptions = [
+    { name: 'SET', value: "SET" },
+    { name: 'GET', value: "GET" },
+  ];
+
+  selectedOperation = this.operationsOptions[0];
+
+  dropdownStaticWeb: any;
+  staticWebOptions: any[] = [
+    {name: 'RPS', benchmark: 'static_web:rps', YLabel : 'Requests per second', scaleField: 'connections', labelsField: 'size', tooltip: "Higher is better.", icon: 'circle-arrow-up'},
+    {name: 'RPS Extrapolated', benchmark: 'static_web:rps-extrapolated', YLabel : 'Requests per second', scaleField: 'connections', labelsField: 'size', tooltip: "Higher is better.", icon: 'circle-arrow-up'},
+    {name: 'Latency', benchmark: 'static_web:latency', YLabel : 'Seconds', scaleField: 'connections', labelsField: 'size', tooltip: "Lower is better.", icon: 'circle-arrow-down'},
+  ];
```

Comment:
this is shared with the server details typescript, and even more: both are copying data from the DB manually, which is loaded by the frontend app, e.g. `ylabel`, `name`, `tooltip` direction etc could be picked up from the benchmarks metadata API endpoint